data/ALI/data.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Artificial Liquid Intelligence Token",
+  "symbol": "ALI",
+  "decimals": 18,
+  "description": "ALI is the native utility token of the Alethea AI Protocol",
+  "website": "https://alethea.ai/",
+  "twitter": "@real_alethea",
+  "tokens": {
+    "ethereum": {
+      "address": "0x6B0b3a982b4634aC68dD83a4DBF02311cE324181"
+    },
+    "goerli": {
+      "address": "0x3ACd26F0b5080C30c066a2055A4254A5BB05F22a"
+    },
+    "base-goerli": {
+      "address": "0x3ACd26F0b5080C30c066a2055A4254A5BB05F22a"
```

Comment:
This token does not have an l2Bridge or an l1Bridge property which means it is not bridgable with op stack bridging. Please add a nobridge and explain how this token is bridge between goerli and base-goerli. E.g. "this is a layer zero token"

Reply:
Hello!

This token is mintable (and burnable) by the L2 StandardBridge 0x4200000000000000000000000000000000000010 in the base goerli network. This permission is set up in the token constructor. The bridge is saved on the token contract as bridge() public property/function.

The l2Bridge() and/or l1Bridge() properties are not part of the IOptimismMintableERC20, neither ILegacyMintableERC20 interfaces. l2Bridge() is present in the impl contract OptimismMintableERC20 but is marked as "legacy" (and yet it is not part of any interface).

Described above makes me confused since it is not really clear what interface the token contract must follow. According to the docs, following the IOptimismMintableERC20 should be enough.
We're not bridging with the LayerZero atm, but with the StandardBridge.

Would renaming the public bridge() property on our token contract into l2Bridge() resolve the issue and make our implementation bridgeable? Do we need to have both l1Bridge() and l2Bridge() present or one is enough (as I understand these are always 0x4200000000000000000000000000000000000010 anyway)?

Please advise – thanks!